### PR TITLE
Added documentation to every class and method in the controllers

### PR DESCRIPTION
**Pull request by Jgregnier**
## Pull Request Description

Added documentation to all of the methods/ classes in our controllers.
## Steps to Test
1. Check for spelling errors.
2. Check that each documentation adequately explains the method its documenting.  
## Link to Feature Ticket
#91, #92

### CODE REVIEW

**Comment by HubDub** [comment]
looks good

File: Controllers/ProductsController.cs [hunk 1]
```diff
@@ -1,33 +1,34 @@
...
 {
     //Class Name: ProductsController
     //Author: Debbie Bourne, Delaine Wendling
     //Purpose of the class: The purpose of this class is to manage the methods that will produce the data and functionality needed for all of the views in the user interface related to products.
-    //Methods in Class: Index(), Types(), TypesList(), Single(), New(), overloaded New(), AddToCart()
+    //Methods in Class: Index(), Types(), TypesList(), Single(), New(), Overloaded New(Product product), AddToCart().
     public class ProductsController : Controller
     {
-        
+        //Bringing in the context from our DB and storing it in a local varialbe named BangazonWebContext.
```
Comment (JamsDuke) [inline Controllers/ProductsController.cs:18]: Variable is misspelled.

**Review by JamsDuke — DISMISSED** [review]
Line 18 in ProductsController has variable misspelled.

**Review by thrashMoneyVillainare — COMMENTED** [review]
nice documenting

**Review by JamsDuke — COMMENTED** [review]
Looking good

File: Controllers/CustomerController.cs [hunk 2]
```diff
@@ -1,44 +1,47 @@
...
             context = ctx;
         }
+
+        //Storing our ActiveCustomer singleton in an private instance.
         private ActiveCustomer singleton = ActiveCustomer.Instance;
+
+        //Method Name: New
+        //Purpose of the Method: Loads new customer form to the view, view wil be static.
+        //Arguments in Method: None.
         public IActionResult New()
```
Comment (delaineackerman) [inline Controllers/CustomerController.cs:32]: We have to add the decorator [HttpGet] above this New method to differentiate it from the post method

**Review by delaineackerman — APPROVED** [review]

**Pull request merged**